### PR TITLE
Updated documentation for Amplify Android Getting Started with REST A…

### PR DESCRIPTION
**Pull request by poojamat**
Updated the docs for restricting API access.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/lib/restapi/fragments/android/getting-started/30_initapi.md [hunk 1]
```diff
@@ -18,8 +18,9 @@ public class MyAmplifyApp extends Application {
         super.onCreate();
 
         try {
-            // Add this line to add the AWSApiPlugin plugin
+            // Add these lines to add the AWSApiPlugin and Auth plugin
             Amplify.addPlugin(new AWSApiPlugin());
+            Amplify.addPlugin(AWSCognitoAuthPlugin());
```
Comment (richardmcclellan) [inline docs/lib/restapi/fragments/android/getting-started/30_initapi.md:23]:
```suggestion
            Amplify.addPlugin(new AWSCognitoAuthPlugin());
```

File: docs/lib/restapi/fragments/android/getting-started/30_initapi.md [hunk 2]
```diff
@@ -18,8 +18,9 @@ public class MyAmplifyApp extends Application {
         super.onCreate();
 
         try {
-            // Add this line to add the AWSApiPlugin plugin
+            // Add these lines to add the AWSApiPlugin and Auth plugin
```
Comment (richardmcclellan) [inline docs/lib/restapi/fragments/android/getting-started/30_initapi.md:21]:
```suggestion
            // Add these lines to add the `AWSApiPlugin` and `AWSCognitoAuthPlugin`
```

File: docs/lib/restapi/fragments/android/getting-started/30_initapi.md [hunk 3]
```diff
@@ -45,8 +47,9 @@ class MyAmplifyApp : Application() {
         super.onCreate()
 
         try {
-            // Add this line to add the AWSApiPlugin plugin
+            // Add these lines to add the AWSApiPlugin and Auth plugin
```
Comment (richardmcclellan) [inline docs/lib/restapi/fragments/android/getting-started/30_initapi.md:50]:
```suggestion
            // Add these lines to add the `AWSApiPlugin` and `AWSCognitoAuthPlugin`
```

File: docs/lib/restapi/fragments/android/getting-started/30_initapi.md [hunk 4]
```diff
@@ -73,8 +76,9 @@ public class MyAmplifyApp extends Application {
         super.onCreate();
 
         try {
-            // Add this line to add the AWSApiPlugin plugin
+            // Add these lines to add the AWSApiPlugin and Auth plugin
```
Comment (richardmcclellan) [inline docs/lib/restapi/fragments/android/getting-started/30_initapi.md:79]:
```suggestion
            // Add these lines to add the `AWSApiPlugin` and `AWSCognitoAuthPlugin`
```

File: docs/lib/restapi/fragments/android/getting-started/30_initapi.md [hunk 5]
```diff
@@ -73,8 +76,9 @@ public class MyAmplifyApp extends Application {
...
             RxAmplify.addPlugin(new AWSApiPlugin());
+            RxAmplify.addPlugin(AWSCognitoAuthPlugin());
```
Comment (richardmcclellan) [inline docs/lib/restapi/fragments/android/getting-started/30_initapi.md:81]:
```suggestion
            RxAmplify.addPlugin(new AWSCognitoAuthPlugin());
```

**Review by div5yesh — APPROVED** [review]
LGTM 🌮

**Review by medelman17 — APPROVED** [review]
Looks good to me; defer to Android team on technical accuracy

**Pull request merged**